R/calc_aggregate_counts.R

```diff
@@ -80,17 +136,23 @@ calc_aggregate_counts <- function(
         ))
 
     if(state){
-        return(state_df)
+        return(harm_df)
     }
 
-    agg_df <- state_df %>%
+    agg_df <- harm_df %>%
```

Comment:
Out of curiosity, why did you name this `harm_df`?? / is `harm` an abbreviation for something?

Reply:
harmonized but yeah i can be clearer with naming and comment